src/JurorsRegistry.ts

```diff
@@ -83,16 +71,19 @@ export function handleJurorBalanceUnlocked(event: JurorBalanceUnlocked): void {
 export function handleJurorTokensAssigned(event: JurorTokensAssigned): void {
   updateJuror(event.params.juror, event)
   createANJMovementForEvent(event.params.juror, REWARD, event.params.amount, event)
+  increaseTotalStaked(event.address, event.params.amount)
```

Comment:
Should we also decrease the Staked ANT on `handleJurorTokensCollected` and `handleJurorSlashed`

Reply:
No, cause after the slash/collect, this function is actually called